script/custom_build_functions.sh

```diff
@@ -1,6 +1,6 @@
 function run_cukes {
   if is_mri_192_plus; then
-    bin/rake acceptance --trace
+    (unset RUBYOPT; bin/rake acceptance --trace)
```

Comment:
Ok I see why that was recommended in the bundler thread, this becomes a question of do we want to keep disabling rubygems on rspec-rails, I mean if rails depends on bundler, and bundler depends on rubygems...

Reply:
Do we now need to add require rubygems instead?

Reply:
I believe it's required by default unless you set `--disable=gems` in `RUBYOPT`.

Reply:
> do we want to keep disabling rubygems on rspec-rails

I don't think we have to. But this is a change we have to address on rspec-dev. 
I tried to readd only rubygems as an option but I get warnings. `unset` seems to be a good option.

Reply:
This is the only build we have an issue with so it doesn't need to be propagated.